src/main/java/org/tdl/vireo/model/repo/impl/SubmissionRepoImpl.java

```diff
@@ -1091,13 +1091,13 @@ public void setColumnOrderingForMonthYearDateFormat(Sort sort, List<String> sqlA
                 value.append("CAST(REPLACE(").append(table).append(".value, ' ', ' 1, ') AS DATE) AS");
             }
 
-            value.append(table).append("_date");
+            value.append(" " + table).append("_date");
```

Comment:
The design here is using string builder to avoid string concatenation.
I suggest instead changing it to add a space on lines 1089 and 1091 like this:
```
value.append("PARSEDATETIME(").append(table).append(".value, 'MMM yyyy') AS ");
```
and
```
value.append("CAST(REPLACE(").append(table).append(".value, ' ', ' 1, ') AS DATE) AS ");
```

Note that the the space is added after ` AS"`.